content/en/mobile_testing/mobile_app_tests/_index.md

```diff
@@ -0,0 +1,148 @@
+    | `{{^is_alert}}`            | Show unless the monitor alerts.                                     |
+    | `{{#is_recovery}}`         | Show when the monitor recovers from `alert`.                          |
+    | `{{^is_recovery}}`         | Show unless the monitor recovers from `alert`.                        |
+    | `{{#is_renotify}}`         | Show when the monitor renotifies.                                   |
+    | `{{^is_renotify}}`         | Show unless the monitor renotifies.                                 |
+    | `{{#is_priority}}`         | Show when the monitor matches priority (P1 to P5).                  |
+    | `{{^is_priority}}`         | Show unless the monitor matches priority (P1 to P5).                |
+
+    Notification messages include the **message** defined in this section and information about the failing locations.
+
+2. Choose team members and services to notify.
+3. Specify a renotification frequency. To prevent renotification on failing tests, leave the option as `Never renotify if the monitor has not been resolved`.
+4. Click **Save & Edit Recording** to save your test configuration and record your mobile app test steps.
+
+For more information, see [Using Synthetic Test Monitors][7].
+
+## Run tests in CI
+
+You can run mobile app tests in a CI pipeline by defining the `mobileApplicationVersionFilePath` option in a [test `synthetics.json` file][13] and a [global configuration `synthetics-ci.config` file][14] as needed. Global configuration file options take precedence over test configuration file options.
+
+In this example, the test `aaa-aaa-aaa` runs with the override application version found in `application/path`.
+
+```json
+// myTest.synthetics.json
+{
+  "tests": [
+    {
+      "id": "aaa-aaa-aaa",
+      "config": {
+        "mobileApplicationVersionFilePath": "application/path"
+      }
+    }
+  ]
+}
+```
+
+Then, run `$ datadog-ci synthetics run-tests --config synthetics-ci.config`.
+
+For more information, see [Continuous Testing and CI/CD][1].
+
+## Permissions
+
+By default, only users with the Datadog Admin and Datadog Standard roles can create, edit, and delete Synthetic mobile app tests. To get create, edit, and delete access to Synthetic mobile app tests, upgrade your user to one of those two [default roles][8].
+
+If you are using the [custom role feature][9], add your user to any custom role that includes `synthetics_read` and `synthetics_write` permissions.
+
+### Restrict access
+
+Access restriction is available for customers using [custom roles][10] on their accounts.
+
+You can restrict access to a mobile app test based on the roles in your organization. When creating a mobile app test, choose which roles (in addition to your user) can read and write your test. 
+
+{{< img src="synthetics/settings/restrict_access_1.png" alt="Set permissions for your test" style="width:70%;" >}}
```

Comment:
This image isn't rendering in the preview, I couldn't find this in the current repo.